composer.json

```diff
@@ -19,17 +19,16 @@
     },
     "require": {
         "php": "^7.2",
-        "cycle/orm": "^1.1",
         "cycle/annotated": "^2.0",
         "cycle/migrations": "^1.0",
+        "cycle/orm": "^1.1",
         "hiqdev/composer-config-plugin": "^1.0@dev",
-        "symfony/console": "^4.3",
         "yiisoft/aliases": "@dev",
-        "yiisoft/friendly-exception": "@dev"
+        "yiisoft/friendly-exception": "@dev",
+        "yiisoft/yii-console": "^3.0@dev"
     },
     "require-dev": {
-        "phpunit/phpunit": "^8.2",
-        "phan/phan": "^2.2"
```

Comment:
now we can remove this section?
```
    "scripts": {
        "phan": "phan --progress-bar -o analysis.txt"	
    },
```

Reply:
Kind of but I think we can get it back when it's fixed in phan repo.

Reply:
https://github.com/phan/phan/pull/3559 has been merged